doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -124,21 +137,25 @@ Computing the mask
    
     The full example described in this section can be found here:
     :doc:`plot_mask_computation.py <../auto_examples/manipulating_visualizing/plot_mask_computation>`.
-    This one can be relevant too:
+    It is also related to this example:
     :doc:`plot_nifti_simple.py <../auto_examples/plot_nifti_simple>`.
 
-If a mask is not given, :class:`NiftiMasker` will try to compute
-one. It is *very important* to take a look at the generated mask, to see if it
-is suitable for your data and adjust parameters if it is not. See the
+If a mask is not specified as an arugment,
+:class:`NiftiMasker` will try to compute
+one from the provided neuroimaging data.
+It is *very important* to verify the quality of the generated mask by
+visualization. This allows to see whether it
+is suitable for your data and intended analyses.
+Alternatively, the mask computation parameters can still be modified. See the
 :class:`NiftiMasker` documentation for a complete list of mask computation
 parameters.
 
-As an example, we will now try to build a mask based on a dataset from
-scratch. The Haxby dataset will be used since it provides a mask that we
-can use as a reference.
+As a first example, we will now automatically build a mask from a dataset.
+We will here use the Haxby dataset because it provides the original mask that
+we can compare the data-derived mask against.
 
-The first step is to generate a mask with default parameters and take
-a look at it.
+The first step is to generate a mask with default parameters and visualize it
+for eye inspection.
```

Comment:
"for" sounds weird, maybe "by" or maybe just stops the sentence as "visualize it".